ansible/deploy_db.yml

```diff
@@ -16,7 +16,7 @@
     - {role: backups, tags: 'backups'}
 
 - name: Postgresql
-  hosts: postgresql
+  hosts: postgresql,pg_standby
```

Comment:
+1 really like this. although it still may have issues when using `--limit=pg0` or something similar